stix_shifter/src/patterns/pattern_objects.py

```diff
@@ -14,9 +14,8 @@ class ComparisonComparators(Enum):
         In,
         Like,
         Matches,
-        IsSubSet,
-        IsSuperSet
```

Comment:
I don't think this needs to be removed as some other datasource may be able to eventually use the SuperSet operator. You've already written the exitPropTestIsSuperset in the parser, so may as well leave this here.